Source/iOS/DolphiniOS/DolphiniOS/UI/Emulation/EAGLView.m

```diff
@@ -0,0 +1,21 @@
+// Copyright 2019 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#import <Foundation/Foundation.h>
+
+#import "EAGLView.h"
+
+@implementation EAGLView
+
+#pragma clang diagnostic push
```

Comment:
There should likely be a comment with this explaining why it needs to exists, as opposed to just fixing the warning.